azure_arc_k8s_jumpstart/docs/aro_terraform.md

```diff
@@ -0,0 +1,232 @@
+# Overview
+
+The following README will guide you on how to use the Azure CLI to deploy an Azure Red Hat OpenShift 4 cluster connected it as an Azure Arc cluster resource.
+
+# Prerequisites
+
+* Create Azure Service Principal (SP) **(This can be done in Cloud Shell as well)**
+
+    To connect the EKS cluster to Azure Arc, Azure Service Principal is required and can be created by:
+
+
+    ```
+    az login
+
+    az ad sp create-for-rbac -n "http://AzureArcK8s" --role contributor
+    ``` 
+
+    Output should look like this:
+    ```
+    {
+       "appId": "XXXXXXXXXXXXXXXXXXXXXXXXXXXX",
+       "displayName": "AzureArcK8s",
+       "name": "http://AzureArcK8s",
+       "password": "XXXXXXXXXXXXXXXXXXXXXXXXXXXX",
+       "tenant": "XXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+    }
+    ```
```

Comment:
- "This can be done in Azure Cloud Shell as well"
- If we want to mention Cloud Shell let's show a screenshot.
- Align with the new "az ad sp create" part [example](https://github.com/likamrat/azure_arc/blob/master/azure_arc_k8s_jumpstart/docs/aks_arm_template.md)

Reply:
Just copy pasted your section